http/websocket.go

```diff
@@ -0,0 +1,30 @@
+package http
+
+import (
+	"net/http"
+
+	"github.com/gorilla/websocket"
+)
+
+type (
+	// Upgrader defines the interface a websocket connection upgrader must satisfy.
```

Comment:
This is the definition for an interface so there's no need to repeat that, maybe something like 

```
// An upgrader is an HTTP connection that is able to upgrade to websocket.
```